### PR TITLE
Update readmes

### PR DESCRIPTION
**Pull request by efritz**

### CODE REVIEW

File: examples/multiple-executors/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+The following variables must be supplied:
+
+- `sourcegraph_external_url`, `sourcegraph_executor_proxy_password`, `queue_name`, `metrics_environment_label`, and `instance_tag`: Analogous to the `executor_*` variables in the `single-executor` example.
+- `resource_prefix`: A prefix unique to each set of compute resources. This prevents collisions between two uses of the `executors` module. We recommend this value be constructed the same way `instance_tag` is constructed.
+- `docker_registry_mirror`: This variable is given the value `"http://${module.docker-mirror.ip_address}:5000"`, which converts the raw external IP address to an address resolvable by the executor instances.
+
+If your deployment environment already has a Docker registry that can be used, only the `executor` submodule must be used (and references to the `networking` and `docker-mirror` modules can be dropped). The Docker registry mirror address can be supplied along with its containing vcp and subnet as pre-existing identifier literals.
```
Comment (Strum355) [inline examples/multiple-executors/README.md:11]:
```suggestion
If your deployment environment already has a Docker registry that can be used, only the `executor` submodule must be used (and references to the `networking` and `docker-mirror` modules can be dropped). The Docker registry mirror address can be supplied along with its containing VPC and subnet as pre-existing identifier literals.
```

Its vpc not vcp, right? :thinking:

**Review by eseliger — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,42 +1,24 @@
...
 
-This example uses the [root module](https://registry.terraform.io/modules/sourcegraph/executors/aws/latest) that provisions a network, a Docker registry mirror, and a set of resources to run _one_ type of executor. To provision more than one type of executor (multiple queues or multiple environments), see the following `multiple-executors` example.
+1. Provisioning executors for use with multiple features (e.g., both [auto-indexing](https://docs.sourcegraph.com/code_intelligence/explanations/auto_indexing) and [server-side batch changes](https://docs.sourcegraph.com/batch_changes/explanations/server_side)), or
+1. Provisioning resources for multiple Sourcegraph instances (e.g., test, prod)
 
-The following variables must be supplied:
+This repository also provides a [root module](https://registry.terraform.io/modules/sourcegraph/executors/aws/latest) combining the executors, network, and docker-mirror resources into an easier to use package.
```
Comment (chrismwendt) [inline README.md:19]: Broken infra image on that page:

CleanShot 2021-11-23 at 11 50 18@2x

File: examples/single-executor/README.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+- `executor_sourcegraph_executor_proxy_password`: The shared executor password defined in Sourcegraph. The value must be the same as the `executors.accessToken` site setting described in [Configuring executors and instance communication](https://docs.sourcegraph.com/admin/deploy_executors#configuring-executors-and-instance-communication).
+- `executor_queue_name`: The name of the target queue to process (e.g., `codeintel`, `batches`).
+- `executor_metrics_environment_label`: The name of the target environment (e.g., `staging`, `prod`). This value must be the same as the `EXECUTOR_METRIC_ENVIRONMENT_LABEL` environment variable as described in [Configuring auto scaling](https://docs.sourcegraph.com/admin/deploy_executors#aws).
+- `executor_instance_tag`: Compute instances are tagged by this value by the key `executor_tag`. We recommend this value take the form `{executor_queue_name}-{executor_metrics_environment_label}`. This value must be the same as `INSTANCE_TAG` as described in [Configuring observability](https://docs.sourcegraph.com/admin/deploy_executors#aws-1).
```
Comment (chrismwendt) [inline examples/single-executor/README.md:13]: Not sure what the `-1` in the link does. It goes to the same heading.

**Review by chrismwendt — APPROVED** [review]

File: examples/single-executor/README.md (hunk 3 above)
Comment (efritz) [reply examples/single-executor/README.md:13]: There should be two Google and two AWS headings, oddly. `aws` should go to the first and `aws-1` to the second.

**Pull request merged**